src/main/io/displayport_msp_bf_compat.c

```diff
@@ -328,28 +328,28 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 
     case SYM_ZERO_HALF_LEADING_DOT:
         return BF_SYM_ZERO_HALF_LEADING_DOT;
+*/
 
     case SYM_AUTO_THR0:
-        return BF_SYM_AUTO_THR0;
+        return 'A';
 
     case SYM_AUTO_THR1:
-        return BF_SYM_AUTO_THR1;
+        return 'T';
 
     case SYM_ROLL_LEFT:
-        return BF_SYM_ROLL_LEFT;
+        return BF_SYM_ROLL;
 
     case SYM_ROLL_LEVEL:
-        return BF_SYM_ROLL_LEVEL;
+        return BF_SYM_ROLL;
 
     case SYM_ROLL_RIGHT:
-        return BF_SYM_ROLL_RIGHT;
+        return BF_SYM_ROLL;
```

Comment:
These roll changes do not remove any ambiguity. You still don't know what it's doing, rendering the OSD element pointless.

You have no way of knowing what direction the roll is.

Reply:
Although the icon itself doesn't say if left or right, it neatly removes the ambiguity of what that number is telling you. It is telling you how many degrees of roll you have. To know if it's left or right, just look at the horizon and any ambiguity is removed. ;-)

Reply:
Not ideal. But I guess it works.

Reply:
I fully agree it isn't ideal, but it is what we have to work with. Thanks!